modules/footer/FooterCTA.tsx

```diff
@@ -73,64 +98,177 @@ const FooterCTA = ( props: Props ): ReactElement => {
       )}
     >
       <div className={classes.container}>
-        <Typography variant="h1" className={classes.typographyH1}>
-          We can&rsquo;t close the technology gender gap without your help.
-        </Typography>
-        <Typography>
-          Join us in making the technology sector in Newfoundland &amp; Labrador as diverse, resilient and creative as the people who live here.
-        </Typography>
-        <div className={classes.containerButtons}>
-          <Grid
-            className={classes.gridContainer}
-            spacing={4}
-            container
-          >
-            {
-              hideVolunteer
-                ? null
-                : (
+        {
+          layoutThreeCols
```

Comment:
Minor, but it may be nice to have breakup the layout three cols and 2 cols into components to reduce the return function length. I don't think it should hold up this PR as I tested it and everything worked great, so I'm approving!

Reply:
Agreed - I think we may ditch the old layout anyways, but this improvement will still make sense. I logged an issue in GitHub so it doesn't get lost: #62